pkg/distributor/distributor.go

```diff
@@ -57,6 +58,13 @@ const (
 
 	labelServiceName = "service_name"
 	serviceUnknown   = "unknown_service"
+	labelLevel       = "level"
```

Comment:
Should we, at least for now, rename that to something like?
```suggestion
	labelLevel       = "level_detected"
```

I'm a bit afraid it could lead to false detections, and then it would even overwrite the "real" detected label from the log line and change that to `level_extracted`, which might break users workflows.

Take this logline as an example:

```
logger=tsdb.loki endpoint=queryData pluginId=loki dsName=grafanacloud-securityops-logs dsUID=grafanacloud-logs traceID=e6560d272e58b8bd94ed0d6da53c5a30 fromAlert=true end=2024-04-02T10:52:26.954369164Z step=30s query="{error=~\".+\"} " queryType=rangestatus=ok level=info
```

It's just an `info` log, but the detection would see the `error=`, which is part of the "parsed" `query` label. Leading to add the `logLevelError` as structured metadata, and the parsed `level` label renamed to `level_extracted`.